script/cmtool.bat

```diff
@@ -98,10 +98,10 @@ goto exit0
 :puppet
 ::::::::::::
 
-:: if "%CM_VERSION%" == "latest" set CM_VERSION=3.8.7
+if "%CM_VERSION%" == "latest" set CM_VERSION=6.3.0
```

Comment:
Rather than hardcoding the version for the latest here, can't we conditional the URL that's being used to point to `puppet-agent-$arch-latest.msi`?